tools/install.pl

```diff
@@ -123,8 +124,19 @@
     );
     say("\r\nInstalling Perl modules... This might take a while.\r\n");
 
-    if ( system("cpanm --installdeps ./tools/. --notest") != 0 ) {
-        die "Something went wrong while installing Perl modules - Bailing out.";
+    # libarchive is not provided by default on macOS, so we set the correct env vars
+    # to successfully compile Archive::Extract::Libarchive and Archive::Peek::Libarchive
+    if ($Config{"osname"} eq "darwin") {
```

Comment:
I think the export calls should be done outside of the if statement -- That way you'll only have one `cpanm` call at the end here, something like that:

```
        if ($Config{"osname"} eq "darwin") {
             `export CFLAGS=\"-I/usr/local/opt/libarchive/include\"`
             `export PKG_CONFIG_PATH=\"/usr/local/opt/libarchive/lib/pkgconfig\"`
        }

        # common code between darwin and linux
        if ( system("cpanm --installdeps ./tools/. --notest") != 0 ) {
            die "Something went wrong while installing Perl modules - Bailing out.";
        }
```

Reply:
That does not work, since `system()` creates a subshell with no/the env vars provided by your profile. I tried that approach before, I don’t like having doubles either but else it does not work sadly

Reply:
I created a fix for this problem, pretty stupid of me to not think of that (not knowing this Perl is no excuse :D)
https://github.com/Bl4Cc4t/LANraragi/commit/e0a306a6205dd09feb5167d6c867da598e75ae27

Reply:
The PR is automatically updated as you push commits, so no need. 👍

Reply:
oh, I did not knew that. So closing my first not necessary D: